src/plugins/intel_cpu/src/nodes/kernels/x64/mlp_kernel.cpp

```diff
@@ -600,11 +621,18 @@ void GateUpCombine::generate() {
 
 void ReduceAdd2bh::generate() {
     if (m_do_reduce2) {
-        Xbyak::Reg64 src0 = abi_param1;
-        Xbyak::Reg64 src1 = abi_param2;
-        Xbyak::Reg64 dst = abi_param3;
-        Xbyak::Reg64 prefetch_dst = abi_param4;
-        Xbyak::Reg64 BN = abi_param5;
+        Xbyak::Reg64 src0 = rdx;
+        Xbyak::Reg64 src1 = r8;
+        Xbyak::Reg64 dst = r9;
+        Xbyak::Reg64 prefetch_dst = r10;
+        Xbyak::Reg64 BN = r11;
```

Comment:
Just for note: from [doc](https://learn.microsoft.com/en-us/cpp/build/x64-software-conventions?view=msvc-160#register-volatility-and-preservation): rax, rcx, rdx, r9-r11 could be used in the callee without protection.